gtsfm/common/keypoints.py

```diff
@@ -216,14 +216,13 @@ def extract_indices(self, indices: np.ndarray) -> "Keypoints":
         """Form subset with the given indices.
 
         Args:
-            indices: indices to extract, as a 1-D vector.
+            indices: Indices to extract, as a 1-D vector.
 
         Returns:
             Subset of data at the given indices.
         """
-
         if indices.size == 0:
-            return Keypoints(coordinates=np.array([]))
+            return Keypoints(coordinates=np.zeros(shape=(0, 2)))
```

Comment:
just curious, how does this matter?

Reply:
All keypoint coordinates need to fit the following on the c++ side:
```
  // The (x, y) coordinates of the features, of shape Nx2.
  Eigen::MatrixX2d coordinates;
```
https://github.com/borglab/gtsam/blob/develop/gtsam/sfm/DsfTrackGenerator.h#L41C1-L42C32